packages/Card/src/components/Title/Title.styles.js

```diff
@@ -1,11 +1,14 @@
 import styled from "styled-components";
 import stylers from "@paprika/stylers";
 import tokens from "@paprika/tokens";
+import Heading from "@paprika/heading";
 
-export const titleStyles = styled.h1`
+export const Title = styled(Heading)`
   box-sizing: border-box;
   color: ${tokens.color.black};
+  font-weight: bold;
   ${stylers.fontSize(0)};
+  line-height: 1.4;
```

Comment:
There's a `styler` for line heights... but why override the `line-height` set by the `<Header>`? 

There might be better ways of vertically aligning, rather than `margin` (maybe flex?)

Reply:
without the line-height it looks like this, i'll try using flex
<img width="262" alt="Screen Shot 2022-04-05 at 9 29 09 AM" src="https://user-images.githubusercontent.com/92059030/161801813-0bac788b-984e-4c87-8622-80bd5b9275ab.png">

Reply:
Oh, I see, that helps.  I this case, I think your line-height makes more sense than flex, but please use the `styler`.